.github/workflows/trymerge.yml

```diff
@@ -84,6 +84,22 @@ jobs:
           set -x
           python3 .github/scripts/comment_on_pr.py "${PR_NUM}" "merge"
 
+      - name: configure aws credentials
+        uses: aws-actions/configure-aws-credentials@v3
+        continue-on-error: true
+        with:
+          role-to-assume: arn:aws:iam::749337293305:role/upload_to_ossci_raw_job_status
```

Comment:
This looks like a wrong AWS account ID.  Should is be `308535385114` to match what is in https://github.com/pytorch-labs/pytorch-gha-infra/pull/421?

Reply:
308535385114 is the ossci AWS account.  749337293305 is the pytorch account.  

The bucket lives in the ossci account